### DISCUSSION TITLE
Dynamic construction of http_url

### DISCUSSION THREAD

**Discussion by espinielli** [githubocto/flat-action#40] [Q&A] [answered]
The data I would like to fetch is a timestamped filename, like `https://example.com/data<date today - 1 day>.csv`

So if today (the date Flat Action is triggered) is the 2nd June 2021 the file is

`https://example.com/data20210601.csv`

Would it be possible to build that url and use it for `http_url`? I.e. having a _variable_/_dynamic_ `http_url`?

**Comment by Wattenberger** [comment, marked as answer]
This sounds like a great feature to add, and this isn't the first time it's come up. It's a larger change than we have time for right now, but we're more than open to PRs!

In the meantime, two (far from ideal!) workarounds I can think of:
- hitting a static endpoint that returns the dynamic content
- embedding the dynamic portion of the url in a GitHub secret that is updated daily
